spec/javascripts/actions/notifications_spec.js

```diff
@@ -0,0 +1,199 @@
+import * as Notification from '../../../app/assets/javascripts/actions/notifications';
+import { types } from '../../../app/assets/javascripts/models/beta/notification';
+import status from 'http-status';
+
+describe('Notifications Actions', () => {
+  describe('sendSuccessNotification', () => {
+    it('dispatches addNotification with the new notification', () => {
+      const newNotification = {
+        id: 42,
+        type: types.SUCCESS,
+        message: 'Success message'
+      }
+
+      const FakeNotification = {
+        createNotification: () => newNotification,
+        types
+      };
+
+      const fakeDispatch = sinon.stub();
+
+      Notification.sendSuccessNotification(newNotification.message)
+        (fakeDispatch, null, { Notification: FakeNotification });
+
+      expect(fakeDispatch).toHaveBeenCalledWith(Notification.addNotification(newNotification));
+    });
+
+    it('dispatches removeNotification after 5000ms to remove notification', () => {
+      const newNotification = {
+        id: 42,
+        type: types.SUCCESS,
+        message: 'Success message'
+      }
+
+      const FakeNotification = {
+        createNotification: () => newNotification,
+        types
+      };
+
+      const fakeDispatch = sinon.stub();
+
+      Notification.sendSuccessNotification(newNotification.message)
+        (fakeDispatch, null, { Notification: FakeNotification });
+
+      setTimeout(() => {
+        expect(fakeDispatch).toHaveBeenCalledWith(Notification.removeNotification(newNotification.id));
```

Comment:
This will never be called in the test since this is not an async test nor it's using `done` callback. But there is a more important thing here: we should avoid using timeouts in tests since it makes our tests dependent on actual time. We should use a faker timer instead: https://sinonjs.org/releases/v1.17.6/fake-timers/